### DISCUSSION TITLE
智斗开发商（进行中...） | LeeDaisen

### DISCUSSION THREAD

**Discussion by giscus** [admin8756/leelaa-blog-comments#15] [Announcements]
# 智斗开发商（进行中...） | LeeDaisen

隔壁房子漏水，无良开发商拒绝承担责任

https://leelaa.cn/posts/ageera

**Comment by readlcy** [comment]
现在解决了吗？

**Reply by admin8756** [reply]
也不知道算不算解决了~反正开发商是没有任何回应的。物业也没什么用。最有效的办法只有起诉了
